mousestyles/classification/Classification.py

```diff
@@ -0,0 +1,72 @@
+from mousestyles import data
+import numpy as np
+from sklearn.grid_search import GridSearchCV
+from sklearn.ensemble import RandomForestClassifier
+from sklearn.preprocessing import StandardScaler
+import pandas as pd 
+
+def RandomForest():
+	df = data.load_all_features()
```

Comment:
Actually, consider taking train_X, train_Y, test_X, test_Y as inputs, this way you don't have to split data into labels and features inside the function. It avoid mistakes that originate from unexpected data.